ext/gensymbol/gensymbol

```diff
@@ -566,6 +581,69 @@ zpotri,
     zgeqrt, zgeqrt2, zgeqrt3, zgemqrt,
     ztpqrt, ztpqrt2, ztpmqrt, ztprfb,
 );
+
+@exlapackobjs2 = (
```

Comment:
This is quite a lot of symbols being added; what happened here?  Were we just missing these?

Reply:
The idea is to support quadruple-precision BLAS/LAPACK with LBT.
The symbols are not standard in quadruple precision; OpenBLAS uses the prefixes `q*` or `x*`, but only for the BLAS symbols and routines with real inputs.
I would like to add a similar paradigm for LAPACK symbols, but I am open to suggestions!

Integrating Fortran code compiled in quadruple precision is straightforward in Julia using the `Float128` data type from `Quadmath.jl`.
However, we need an efficient way to compile our library with BLAS/LAPACK libraries that have dedicated symbols for quadruple precision.
LBT is optimal because we can add a dedicated backend compiled in quadruple precision at runtime.

Reply:
No, I was planning to compile Reference-LAPACK in quadruple precision.
A simple way to do this is to use the compiler-dependent flag `-freal-8-real-16` with `gfortran` and some macros for the symbols.

The goal is to use it in libHSL, which is more or less similar to SuiteSparse and has some of the most robust linear solvers, such as `MA27` and `MA57`, available in quadruple precision.
I can easily call them later in Julia with wrappers like [this](https://github.com/JuliaSmoothOptimizers/CUTEst.jl/blob/main/src/libcutest.jl#L1253-L1256).
CUTEst doesn’t have any dependencies, unlike libHSL, which requires BLAS / LAPACK.

Reply:
I see. And so you’re trying to start a new convention with this naming scheme, or is there already a naming scheme for this and you’re conforming to that?

Reply:
For the `q*` symbols, I followed what was initially established by GotoBLAS, so I’m not creating a new convention here; instead, I’m following one that already exists informally. I proposed splitting this PR into two parts, one of which would include only the quadruple symbols.

I’ll contact Julien Langou, one of the developers of Reference-LAPACK, to confirm this informal convention, and then we can decide how to proceed.